### PR TITLE
Update determine-reason-pod-failure.md

### PR DESCRIPTION
**Pull request by stewart-yu**
1、When run `kubectl apply/create -f `, we should better use origin file
2、fix shadow for commans.

Allow edits from maintainers checkbox

### CODE REVIEW

**Comment by stewart-yu** [comment]
/assign @tengqm @chenopis 
for review

File: content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ the container starts.
 
 1. Create a Pod based on the YAML configuration file:
 
-       kubectl create -f https://k8s.io/docs/tasks/debug-application-cluster/termination.yaml
+        kubectl create -f https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/tasks/debug-application-cluster/termination.yaml
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: Why this change?

File: content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md [hunk 2]
```diff
@@ -47,13 +47,13 @@ the container starts.
 
 1. Display information about the Pod:
 
-       kubectl get pod termination-demo
+        kubectl get pod termination-demo
```
Comment (tengqm) [inline content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: need to format lines like this as code explicitly.

File: content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md (hunk 1 above)
Comment (stewart-yu) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: The same as  PR[https://github.com/kubernetes/website/pull/6435](https://github.com/kubernetes/website/pull/6435)

File: content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md (hunk 2 above)
Comment (stewart-yu) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: In general, using 
'''shell   
'''
But there have numlist, we should format like that. Besides, If we format there explicitly, doesn't work. Look https://deploy-preview-8837--kubernetes-io-master-staging.netlify.com/docs/tasks/debug-application-cluster/determine-reason-pod-failure/, it's correctly.

Comment (tengqm) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: Checked the preview. It is okay.

File: content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: I am curious why the old url doesn't work.

Comment (zhangxiaoyu-zidif) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: the old url works in my env... curious.

Comment (steveperry-53) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: Please don't change URL of the yaml file. If this isn't working, then we need to get the yaml file in the correct directory.

Comment (stewart-yu) [reply content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md]: fixed, PTAL
